resdk/resources/sample.py

```diff
@@ -21,6 +21,21 @@ def get_bam(self):
         """Return ``bam`` object on the sample."""
         return self.data.get(type='data:alignment:bam:')
 
+    def get_primary_bam(self, bam=True):
+        """Return ``primary bam`` object on the sample.
+
+        If the ``primary bam`` object is not present the function will
+        try to return a ``bam`` object if bam=True.
+
+        """
+        try:
+            return self.data.get(type='data:alignment:bam:primary:')
+        except LookupError:
+            if bam:
+                return self.get_bam()
```

Comment:
I don't like this as ``get_primary_bam`` name implies that primary bam will be returned.